### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by barrem01** [sproctor/php-calendar#124]
I just followed the instructions at install.php.  I left most values at the default, gave the script my mysql admin username and password and asked it to create everything.

I created a new calendar, and made it the default. When I attempted to add an item to the calendar by clicking on the "+" link I get the error:

```
Error in get_fields

Table 'calendar.phpc_fields' doesn't exist

SQL query: SELECT `name`, `required`, `format`, `cid`, `fid`
FROM `phpc_fields`
WHERE `phpc_fields`.`cid` = '3'

Backtrace

    phpcdatabase.class.php(372): db_error('Error in get_fields', 'SELECT `name`, `required`, `format`, `cid`, `fid` FROM `phpc_fields` WHERE `phpc_fields`.`cid` = '3' ')
    phpccalendar.class.php(160): get_fields('3')
    event_form.php(140): get_fields()
    event_form.php(28): display_form()
    calendar.php(704) : eval()'d code(1): event_form()
    calendar.php(704): eval(...)
    calendar.php(627): do_action()
    index.php(102): display_phpc()
```

It doesn't look like a phpc_fields table was created:

```
mysql> use calendar
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Database changed
mysql> SELECT `name`, `required`, `format`, `cid`, `fid` FROM `phpc_fields` WHERE `phpc_fields`.`cid` = '2';
ERROR 1146 (42S02): Table 'calendar.phpc_fields' doesn't exist
mysql> show tables;
+----------------------+
| Tables_in_calendar   |
+----------------------+
| phpc_calendar_fields |
| phpc_calendars       |
| phpc_categories      |
| phpc_config          |
| phpc_events          |
| phpc_groups          |
| phpc_logins          |
| phpc_occurrences     |
| phpc_permissions     |
| phpc_user_groups     |
| phpc_users           |
+----------------------+
11 rows in set (0.00 sec)
```

Any advice?

**Comment by sproctor** [comment]
Download the most recent version from https://github.com/sproctor/php-calendar/releases

**Comment by barrem01** [comment]
Thanks for your very quick response.
I've downloaded and uncompressed 2.0.8.  (BTW, I got the old version from https://github.com/sproctor/php-calendar/archive/master.zip - the "Download ZIP" button on github.)
deleted all files in my old calendar installation directory and copied the new files and directories (including the hidden ones) into the installation directory, touched and change permissions on config.php, and dropped the calendar database in mysql.
Now when I run install.php I choose the existing calendar user and ask it to create the databases. Eventually I get to
...
PHP Calendar

Welcome to the PHP Calendar installation process.

Connecting to SQL server localhost with user calendar.

Selecting database calendar

Writing config file "/var/www/oso/calendar/config.php".

Creating table "SQL_PREFIXevents".

Creating table "SQL_PREFIXoccurrences".

Creating table "SQL_PREFIXusers".

Creating table "SQL_PREFIXgroups".

Creating table "SQL_PREFIXuser_groups".
Step 3: Database Created

Creating table "SQL_PREFIXlogins".

Logins table updated.

Creating table "SQL_PREFIXpermissions".

Creating table "SQL_PREFIXcalendars".

Creating table "SQL_PREFIXcategories".

Creating table "SQL_PREFIXversion".

Config file created at "/var/www/oso/calendar/config.php"

Calendars database created
...
which eventually results in 

...
Software Error
Message:

Error reading options
Table 'calendar.phpc_calendars' doesn't exist
SQL query: INSERT INTO phpc_calendars
(`cid`) VALUE (1)

Backtrace

    /var/www/oso/calendar/install.php:690 - soft_error
    /var/www/oso/calendar/install.php:657 - db_error
    /var/www/oso/calendar/install.php:136 - add_calendar
...
with config.php = 
..
$ more config.php
<?php
define('SQL_HOST',     'localhost');
define('SQL_USER',     'calendar');
define('SQL_PASSWD',   <redacted>);
define('SQL_DATABASE', 'calendar');
define('SQL_PREFIX',   'phpc_');
define('SQL_TYPE',     'mysqli');
?>
...
and the mysql calendar tables = 
...
mysql> show tables;
+-----------------------+
| Tables_in_calendar    |
+-----------------------+
| SQL_PREFIXcalendars   |
| SQL_PREFIXcategories  |
| SQL_PREFIXevents      |
| SQL_PREFIXgroups      |
| SQL_PREFIXlogins      |
| SQL_PREFIXoccurrences |
| SQL_PREFIXpermissions |
| SQL_PREFIXuser_groups |
| SQL_PREFIXusers       |
| SQL_PREFIXversion     |
+-----------------------+
10 rows in set (0.00 sec)
...

**Comment by sproctor** [comment]
Sorry about that. I forgot that I hadn't released a new version that fixed the installer issues from PHP 5.5. I just tagged 2.0.9 for release. https://github.com/sproctor/php-calendar/releases/tag/v2.0.9 This should fix your issue. PHP 5.5 changed the behavior of defines in a file included from inside a function.

**Comment by barrem01** [comment]
Success!  Thank you very much for your quick response.  I look forward to working with your calendar.
